Formula/gnirehtet.rb

```diff
@@ -1,12 +1,22 @@
 class Gnirehtet < Formula
   desc "Reverse tethering tool for Android"
   homepage "https://github.com/Genymobile/gnirehtet"
-  url "https://github.com/Genymobile/gnirehtet/archive/v2.5.tar.gz"
-  sha256 "2b55b56e1b21d1b609a0899fe85d1f311120bb12b04761ec586187338daf6ec5"
   license "Apache-2.0"
   revision 1
   head "https://github.com/Genymobile/gnirehtet.git", branch: "master"
 
+  stable do
+    url "https://github.com/Genymobile/gnirehtet/archive/v2.5.tar.gz"
+    sha256 "2b55b56e1b21d1b609a0899fe85d1f311120bb12b04761ec586187338daf6ec5"
+
+    # Fix compilation issue with rust 1.64.0+
+    # upstream PR reference, https://github.com/Genymobile/gnirehtet/pull/478
```

Comment:
The direct `git apply` has some conflict, I just re-generate the cargo.lock from 2.5 release artifact.